plugin-server/src/cdp/templates/index.ts

```diff
@@ -38,6 +39,7 @@ export const HOG_FUNCTION_TEMPLATES_DESTINATIONS: HogFunctionTemplate[] = [
     tiktokAdsTemplate,
     snapchatAdsTemplate,
     linearTemplate,
+    githubTemplate,
```

Comment:
We may want to remove that from here. Imo this is extremely confusing that it is available in the `Data pipelines` tab as:
- you can't filter for events this is intended for (as this is an internal event which should trigger that),
- this internal event has special fields which are not available in normal event so reading the template is confusing,
- to my knowledge it just doesn't work. Seems like `Data pipelines` ignores internal events.

Reply:
Won't do that in this PR. It would require major changes for this to work and removing that from this file does not fix the issue